src/ASM/ASMs2D.h

```diff
@@ -240,8 +240,9 @@ class ASMs2D : public ASMstruct, public ASM2D
   //! \brief Returns a matrix with nodal coordinates for an element.
   //! \param[in] iel Element index
   //! \param[out] X 3\f$\times\f$n-matrix, where \a n is the number of nodes
+  //! \param[in] forceItg If true return integration basis element coordinates
   //! in one element
```

Comment:
The added doxy line is misplaced, as the "in one element" belongs to the X-parameter description.

Reply:
fixed.